<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentityEditWidget</class>
 <widget class="QWidget" name="IdentityEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Real Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="realName">
           <property name="whatsThis">
            <string>The &quot;Real Name&quot; is shown in /whois.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Nicknames</string>
         </property>
         <layout class="QHBoxLayout" name="_3">
          <item>
           <widget class="QListWidget" name="nicknameList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="_4">
            <item>
             <widget class="QPushButton" name="addNick">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Add Nickname</string>
              </property>
              <property name="text">
               <string>&amp;Add...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/16x16/actions/oxygen/16x16/actions/list-add.png</normaloff>:/16x16/actions/oxygen/16x16/actions/list-add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteNick">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Remove Nickname</string>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/16x16/actions/oxygen/16x16/actions/edit-delete.png</normaloff>:/16x16/actions/oxygen/16x16/actions/edit-delete.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="renameNick">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Rename Identity</string>
              </property>
              <property name="text">
               <string>Re&amp;name...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/16x16/actions/oxygen/16x16/actions/edit-rename.png</normaloff>:/16x16/actions/oxygen/16x16/actions/edit-rename.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="_5">
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="nickUp">
                <property name="toolTip">
                 <string>Move upwards in list</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/16x16/actions/oxygen/16x16/actions/go-up.png</normaloff>:/16x16/actions/oxygen/16x16/actions/go-up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="nickDown">
                <property name="toolTip">
                 <string>Move downwards in list</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/16x16/actions/oxygen/16x16/actions/go-down.png</normaloff>:/16x16/actions/oxygen/16x16/actions/go-down.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>124</width>
                <height>76</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="awayTab">
      <attribute name="title">
       <string>A&amp;way</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Default Away Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="_6">
            <item row="0" column="1">
             <widget class="QLineEdit" name="awayNick">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Nick to be used when being away</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="awayReason">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Default away reason</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="awayNickLabel">
              <property name="text">
               <string>Away Nick:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="awayReasonLabel">
              <property name="text">
               <string>Away Reason:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="detachAwayEnabled">
         <property name="toolTip">
          <string>Set away when all clients have detached from the core</string>
         </property>
         <property name="title">
          <string>Away On Detach</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Away Reason:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="detachAwayReason"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="autoAwayEnabled">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Not implemented yet</string>
         </property>
         <property name="title">
          <string>Away On Idle</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="_7">
            <item>
             <widget class="QLabel" name="autoAwayLabel_1">
              <property name="text">
               <string>Set away after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="autoAwayTime"/>
            </item>
            <item>
             <widget class="QLabel" name="autoAwayLabel_2">
              <property name="text">
               <string>minutes of being idle</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="_8">
            <item row="0" column="0">
             <widget class="QCheckBox" name="autoAwayReasonEnabled">
              <property name="text">
               <string>Away Reason:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="autoAwayReason">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="_9">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Ident:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ident">
           <property name="whatsThis">
            <string>The &quot;ident&quot; is part of your hostmask and, together with your host, uniquely identifies you within the IRC network.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Messages</string>
         </property>
         <layout class="QVBoxLayout" name="_10">
          <item>
           <layout class="QGridLayout" name="_11">
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Part Reason:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="partReason"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="kickReason"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Quit Reason:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="quitReason"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Kick Reason:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="keyAndCertSettings">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageNoSsl">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>You need an SSL Capable Client to edit your Cores SSL Key and Certificate</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageUnsecure">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Warning: you are not connected with a secured connection to the Quassel Core!
Proceeding will cause an unencrypted transfer of your SSL Key and SSL Certificate!</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="continueUnsecured">
                  <property name="text">
                   <string>Continue</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageEditSsl">
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="sslKeyGroupBox">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string>Use SSL Key</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Key Type:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="keyTypeLabel">
                  <property name="text">
                   <string>No Key loaded</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="clearOrLoadKeyButton">
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="sslCertGroupBox">
             <property name="title">
              <string>Use SSL Certificate</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Organisation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="certOrgLabel">
                    <property name="text">
                     <string>No Certificate loaded</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>CommonName:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="certCNameLabel">
                    <property name="text">
                     <string>No Certificate loaded</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_14">
                <item>
                 <widget class="QPushButton" name="clearOrLoadCertButton">
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>realName</tabstop>
  <tabstop>nicknameList</tabstop>
  <tabstop>addNick</tabstop>
  <tabstop>deleteNick</tabstop>
  <tabstop>renameNick</tabstop>
  <tabstop>nickUp</tabstop>
  <tabstop>nickDown</tabstop>
  <tabstop>awayNick</tabstop>
  <tabstop>awayReason</tabstop>
  <tabstop>detachAwayEnabled</tabstop>
  <tabstop>detachAwayReason</tabstop>
  <tabstop>autoAwayEnabled</tabstop>
  <tabstop>autoAwayTime</tabstop>
  <tabstop>autoAwayReasonEnabled</tabstop>
  <tabstop>autoAwayReason</tabstop>
  <tabstop>ident</tabstop>
  <tabstop>kickReason</tabstop>
  <tabstop>partReason</tabstop>
  <tabstop>quitReason</tabstop>
  <tabstop>continueUnsecured</tabstop>
  <tabstop>clearOrLoadKeyButton</tabstop>
  <tabstop>clearOrLoadCertButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
